src/components/OptionsSelector.js

```diff
@@ -59,6 +60,10 @@ const propTypes = {
 
     // Whether to show the title tooltip
     showTitleTooltip: PropTypes.bool,
+
+    // The ref to the search input
+    // eslint-disable-next-line react/forbid-prop-types
```

Comment:
Surely we will use this enough times that a reusable prop is better than `PropTypes.object.isRequired` ?